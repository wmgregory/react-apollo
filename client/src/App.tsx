import React from 'react';

import './App.css';

import { Header } from './components';

const App: React.FC = () => {
  return (
    <div className='App'>
      <Header title='React Apollo demo' />
    </div>
  );
};

export default App;
