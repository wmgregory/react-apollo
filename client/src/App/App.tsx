import React from 'react';
import { ApolloProvider } from '@apollo/react-hooks';

import './App.css';

import { Header } from '../components';
import { client } from '../services/client';

export const App: React.FC = () => {
  return (
    <ApolloProvider client={client}>
      <div className="App">
        <Header title="React Apollo demo" />
      </div>
    </ApolloProvider>
  );
};
