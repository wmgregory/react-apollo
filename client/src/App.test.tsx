import React from 'react';

import { shallow } from 'enzyme';

import App from './App';

it('renders without crashing', () => {
  // const div = document.createElement('div');
  // ReactDOM.render(<App />, div);
  // ReactDOM.unmountComponentAtNode(div);

  const wrapper = shallow(<App />);
  expect(wrapper).toBeDefined();
  expect(wrapper).toMatchSnapshot();
});
