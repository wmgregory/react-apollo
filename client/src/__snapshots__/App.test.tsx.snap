// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
"<div className=\\"App\\">
  <Header title=\\"React Apollo demo\\" />
</div>"
`;
